cypress/integration/pages/onDemandRadioEpisode/testsForCanonicalOnly.js

```diff
@@ -0,0 +1,78 @@
+import appConfig from '../../../../src/server/utilities/serviceConfigs';
+import envConfig from '../../../support/config/envs';
+import getEmbedUrl from './helpers';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRunForCanonicalOnly = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRunForCanonicalOnly to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
+  service,
+  pageType,
+  variant,
+}) =>
+  describe(`testsThatFollowSmokeTestConfigForCanonicalOnly for ${service} ${pageType}`, () => {
+    describe('Audio Player', () => {
+      const { lang } = appConfig[service][variant];
+      let embedUrl;
+
+      beforeEach(() => {
+        cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
+          embedUrl = getEmbedUrl(body, lang);
+        });
+      });
+
+      it('should be rendered', () => {
+        cy.get(`iframe[src*="${embedUrl}"]`).should('be.visible');
+      });
+
+      it('embed URL should be reachable', () => {
+        cy.testResponseCodeAndType(embedUrl, 200, 'text/html');
+      });
+    });
+
+    describe('Brand image visible above 400, not visible below 400', () => {
+      // This is how you can test more than one viewport, but for speed we probably want to just test
+      // on one viewport below breakpoint and one default above breakpoint
+      // We will not need to use the array then, and can just use two cy.viewport commands
+      const sizesBelowBreakpoint = ['iphone-6'];
+
+      sizesBelowBreakpoint.forEach(size => {
+        // make assertions on the image using
+        // an array of different viewports
+        it(`Should display image on ${size} screen`, () => {
+          if (Cypress._.isArray(size)) {
+            cy.viewport(size[0], size[1]);
+          } else {
+            cy.viewport(size);
+          }
+
+          cy.visit(`${Cypress.env('currentPath')}`);
+          // Just using hamburger menu button as an example until the image is on test
+          cy.get('nav').find('button').should('be.visible');
+        });
+      });
+    });
```

Comment:
This test can be moved to tests.js (since it is duplicated for AMP and Canonical

Reply:
Moved